swagger.yaml

```diff
@@ -3371,7 +3371,7 @@ definitions:
         description: Installer version.
       installation_disk_path:
         type: string
-        description: Host installation path.
+        description: Contains the inventory disk id to install on. This field should actually be called installationDiskID, the field has been repurposed.
```

Comment:
Too wordy and unclear. Just say 'The ID of the selected installation disk'

Reply:
I requested this description - the field is called "path" but we changed its purpose to "id" and kept the old name.

I think this warrants a clear explanation because it's not intuitive for anyone not familiar with what happened.

A disk has a `path`, `by-path`, `by-id` and `id` (which is confusing enough), a field called installation_disk_path all of a sudden contains the disk ID rather than the path - this adds even more confusion. This explanation clears it up somewhat.

Also the description was meant to be verbose, I don't see a reason saving on words here

Reply:
Please let me know what's the final decision